entrypoint.sh

```diff
@@ -6,7 +6,7 @@ TRACEE_RULES_EXE=${TRACEE_RULES_EXE:="/tracee/tracee-rules"}
 # handle sub commands
 if [ "$1" = "trace" ]; then
 	shift
-	$TRACEE_EBPF_EXE $@
+	exec $TRACEE_EBPF_EXE $@
```

Comment:
Reason for adding: If Tracee is running inside a container and I try to do the following:
```
docker kill --signal=SIGINT tracee
```
from outside of the container, the SIGINT is not delivered to the Tracee. 

Let me know if there's a better approach than this.

Reply:
Created https://github.com/aquasecurity/tracee/issues/736 to track it